src/components/Stakeholders/Add.js

```diff
@@ -0,0 +1,357 @@
+                  value={data.org_name}
+                  onKeyDown={handleEnterPress}
+                />
+              </FormControl>
+            )}
+            <FormControl xs={12} sm={6} className={classes.root}>
+              <TextField
+                error={!!errors?.email}
+                helperText={errors.email}
+                className={classes.textField}
+                label="Email"
+                variant="outlined"
+                name="email"
+                type="email"
+                onChange={handleChange}
+                value={data.email}
+              />
+            </FormControl>
+            <FormControl xs={12} sm={6} className={classes.root}>
+              <TextField
+                error={!!errors?.phone}
+                helperText={errors.phone}
+                className={classes.textField}
+                label="Phone"
+                variant="outlined"
+                name="phone"
+                type="tel"
+                onChange={handleChange}
+                value={data.phone}
+                onKeyDown={handleEnterPress}
+              />
+            </FormControl>
+            <FormControl xs={12} sm={6} className={classes.root}>
+              <TextField
+                className={classes.textField}
+                label="Website"
+                variant="outlined"
+                name="website"
+                type="url"
+                onChange={handleChange}
+                value={data.website}
+                onKeyDown={handleEnterPress}
+              />
+            </FormControl>
+            <FormControl xs={12} sm={6} className={classes.root}>
+              <TextField
+                className={classes.textField}
+                label="Logo URL"
+                variant="outlined"
+                name="logo_url"
+                type="url"
+                onChange={handleChange}
+                value={data.logo_url}
+                onKeyDown={handleEnterPress}
+              />
+            </FormControl>
+            <FormControl xs={12} sm={6} className={classes.root}>
+              <TextField
+                className={classes.textField}
+                label="Map"
+                variant="outlined"
+                name="map"
+                placeholder="map_address"
+                onChange={handleChange}
+                value={data.map}
+                onKeyDown={handleEnterPress}
+              />
+            </FormControl>
+          </Grid>
+          <Grid container justify-content="space-between">
+            <FormControl xs={12} sm={6} className={classes.root}>
+              <FormLabel id="relation">Relationship</FormLabel>
+              <RadioGroup
+                aria-labelledby="radio-buttons-group-label"
+                name="relation"
+                value={data.relation}
+                className={classes.radioGroup}
+                onKeyDown={handleEnterPress}
+              >
+                <FormControlLabel
```

Comment:
Is there a reason the two radio options are formatted differently?

Reply:
Are you referring to a radio option on a different page?

Reply:
These two `FormControlLabel`s have different syntax:
```
                <FormControlLabel
                  control={
                    <Radio
                      label="Parent"
                      value="parents"
                      onChange={handleChange}
                    />
                  }
                  label="Parent"
                />
                <FormControlLabel
                  label="Child"
                  value="children"
                  control={<Radio />}
                  onChange={handleChange}
                />
```
The result is the same (I assume), but they should be consistent.